src/30_Advanced/Using_the_AMP_URL_API.html

```diff
@@ -0,0 +1,42 @@
+<!---{
+  "hideCode": true,
+  "hidePreview": true
+}--->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async custom-element="amp-iframe" src="https://cdn.ampproject.org/v0/amp-iframe-0.1.js"></script>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+<!--
+  #### Introduction
+
+  The [AMP URL API](https://developers.google.com/amp/cache/use-amp-url)
+  retrieves the matching AMP HTML URLs for a given list of URLs. Apps can use
+  this if they want to load the fast AMPs instead of the original URL, for example,
+  when opening external links.
+
+  #### See how it works
+
+  Press execute to perform a live query against the AMP URL API. Change the input 
+  query and try different links. 
+
+  <amp-iframe width="auto" height="500"
```

Comment:
I noticed the json inside the iframe for the request and the json displayed as response have different fonts.